packages/core-app-api/src/apis/implementations/auth/microsoft/MicrosoftAuth.test.ts

```diff
@@ -79,6 +79,16 @@ describe('MicrosoftAuth', () => {
         'Requested access token with scopes from multiple Azure resources: one-resource, other-resource. Access tokens can only have a single audience.',
       );
     });
+
+    it('succeeds when requesting multiple scopes for the same resource', async () => {
```

Comment:
Should we also add a test case for something like the following?
```ts
const accessTokenPromise = microsoftAuth.getAccessToken(['one-scope', 'other-scope']); // or getAccessToken('one-scope other-scope')
```

Reply:
to be clear, you're emphasizing testing with scopes which implicitly belong to the microsoft graph resource?

Reply:
Yes.

Reply:
I'm into it